content/ja/handbook/marketing/corporate-communications/incident-communications-plan.md

```
@@ -0,0 +1,176 @@
+
+#### ティア 4 - イベント
+
+ティア 4 のイベントとは、[セキュリティインシデント](/handbook/security/security-operations/sirt/sec-incident-response/) や Zendesk 経由のカスタマーサポート要請のように、すでにプロセスが確立されているものです。ティア 4 の場合は、確立された会社プロセスに従ってください。
+
+#### ティア 3 - 迅速な対応
+
+ティア 3 は、業界の変化、競合の動き、自社に関するニュース、その他コメントや意見、ソートリーダーシップを提供する機会に対処するために必要な迅速な対応です。ティア 3 の場合は、[迅速な対応指針に従ってください](/handbook/marketing/#-marketing-rapid-response-process)。
+
+#### ティア 2 - インシデント（プロセスは下記）
+
+ティア 2 のインシデントとは、現在進行中の問題、[グレーゾーンのコミュニケーション](/handbook/values/#religion-and-politics-at-work)、または計画された機微な企業コミュニケーションで、顧客・チームメンバー・市場の信頼を失う可能性のあるものです。インシデントは短期的な影響ですが、リスクは高いものです。例として、採用方針、歓迎されない製品変更の実装、経営陣の交代、[会社の価値観](/handbook/values/#credit)に対する不誠実さなどが挙げられます。インシデントの基準を満たす場合は、**[下記のプロセス](#tier-2-process)を開始してください**。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)